SU2_CFD/src/variables/CIncEulerVariable.cpp

```diff
@@ -113,6 +114,8 @@ CIncEulerVariable::CIncEulerVariable(su2double pressure, const su2double *veloci
   Delta_Time.resize(nPoint) = su2double(0.0);
   Lambda.resize(nPoint) = su2double(0.0);
   Sensor.resize(nPoint) = su2double(0.0);
+  Streamwise_Periodic_RecoveredPressure.resize(nPoint)    = su2double(0.0);
+  Streamwise_Periodic_RecoveredTemperature.resize(nPoint) = su2double(0.0);
 
```

Comment:
if (streamwise_periodic) ?

Reply:
uff yeah for sure .. two nPoint allocations are not really great. it is now 
```c++
  if (config->GetKind_Streamwise_Periodic() != NONE) {
    Streamwise_Periodic_RecoveredPressure.resize(nPoint) = su2double(0.0);
    if (config->GetStreamwise_Periodic_Temperature())
      Streamwise_Periodic_RecoveredTemperature.resize(nPoint) = su2double(0.0);
  }
```

Reply:
It would not make a whole in SU2's guts (as my grandmother would say) but if it can be avoided with a couple lines...